lib/Signaling/Messages.php

```diff
@@ -129,7 +129,8 @@ public function getAndDeleteMessages(string $sessionId): array {
 		$query->select('*')
 			->from('talk_internalsignaling')
 			->where($query->expr()->eq('recipient', $query->createNamedParameter($sessionId)))
-			->andWhere($query->expr()->lte('timestamp', $query->createNamedParameter($time)));
+			->andWhere($query->expr()->lte('timestamp', $query->createNamedParameter($time)))
+			->orderBy('id');
```

Comment:
```suggestion
			->orderBy('id', 'ASC');
```

Reply:
ASC is the default to Doctrine DBAL:
https://github.com/nextcloud/3rdparty/blob/master/doctrine/dbal/src/Query/QueryBuilder.php#L1243
Follow as default at the latest version of DBAL:
https://github.com/doctrine/dbal/blob/3.6.x/src/Query/QueryBuilder.php#L1277

And also to
MySQL:
https://dev.mysql.com/doc/refman/8.0/en/sorting-rows.html
PostgreSQL:
https://www.postgresql.org/docs/current/queries-order.html
SQLite:
https://www.sqlitetutorial.net/sqlite-order-by/
And Oracle:
https://www.oracletutorial.com/oracle-basics/oracle-order-by/

Reply:
Explicit is better :)

Reply:
https://github.com/nextcloud/spreed/issues/9077